File: backend/CoverMe.Backend.Core/Models/Settings/CoverageSettings.cs
namespace CoverMe.Backend.Core.Models.Settings;

public class CoverageSettings
{
    public string ProjectsFilter { get; set; } = "*.Tests.csproj";
    public string TestsFilter { get; set; } = "FullyQualifiedName~.Tests";
    public string CoverageFilter { get; set; } = "-:module=testhost;-:module=*.Tests";
    public bool HideAutoProperties { get; set; } = true;
}

File: .old/native-1/src/rider/main/kotlin/com/jetbrains/rider/plugins/coverme/models/settings/CoverageSettings.kt
package com.jetbrains.rider.plugins.coverme.models.settings

import kotlinx.serialization.Serializable

@Serializable
data class CoverageSettings(
    var testsProjectFilter: String,
    var testsFilter: String,
    var coverageFilter: String,
    var hideAutoProperties: Boolean
)